### ISSUE TITLE
[Accessibility issue] Vignette lacks alt= in plot

### ISSUE THREAD

**Issue by MichaelChirico** [tidymodels/rsample#411]
We have a tool that flags images that lack an `alt=` attribute. See https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices -- these are essential for conveying the image contents to users on screen readers.

You can add these to your vignettes by using the recent-ish `fig.alt` chunk option in knitr; see https://www.rstudio.com/blog/knitr-fig-alt/.

The vignette at issue is https://github.com/tidymodels/rsample/blob/main/vignettes/Working_with_rsets.Rmd

Unfortunately the lines are flagged in the HTML output (lines 554), which won't correspond exactly to the .Rmd file lines, but I assume if you're familiar with the vignette it will be clear which images need it.

**Comment by hfrick** [comment]
@MichaelChirico thank you for reporting that! we have a company off-site atm but I'll fix this when we're back.
